Jenkinsfile_CNP

```diff
@@ -8,7 +8,7 @@ import uk.gov.hmcts.contino.GithubAPI
 def type = "java"
 def product = "civil"
 def component = "service"
-def ccdBranch = "master"
+def ccdBranch = "feat/CIV-5486_case_file_view"
```

Comment:
As standard practice I'll mark the PR as "Request changes" to prevent this from being merged to master by mistake, fully aware that you're doing it to see the build turn green.

Reply:
exactly, also need to ensure feat/CIV-5486_case_file_viewis merged before this civil PR

Reply:
Namely, this PR: https://github.com/hmcts/civil-ccd-definition/pull/2502